### PR TITLE
add us core patient api

### PR DESCRIPTION
**Pull request by eclendenning-va**

### CODE REVIEW

File: us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+public interface PatientApi {
+  @Operation(
+      summary = "Patient Read",
+      description = "https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html",
```
Comment (jblefkove-va) [inline us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java:19]: should still be the `build.fhir.org` link

File: us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      tags = {"Patient"})
+  @GET
+  @Path("Patient/{id}")
+  @ApiResponse(
+      responseCode = "200",
+      description = "Record found",
+      content =
+          @Content(
+              mediaType = "application/json+fhir",
```
Comment (jblefkove-va) [inline us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java:28]: hmmm... shouldn't this be `fhir+json`? I think this is backwards for `AllergyIntoleranceApi` and `ConditionApi` as well

Comment (jhulbert-va) [reply us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java:28]: It should be `fhir+json`, yeah. :big_oof:

Comment (eclendenning-va) [reply us-core-r4/src/main/java/gov/va/api/health/uscorer4/api/PatientApi.java:28]: I gotchu

**Pull request merged**